yt/visualization/plot_modifications.py

```diff
@@ -961,11 +961,12 @@ def __init__(
         self.factor = _validate_factor_tuple(factor)
         self.clim = clim
         self.take_log = take_log
-        self.plot_args = {
-            "colors": "black",
-            "linestyles": "solid",
-            **(plot_args or {}),
-        }
+
+        self.plot_args = {**(plot_args or {})}
+        if "colors" not in self.plot_args and "cmap" not in self.plot_args:
+            self.plot_args["colors"] = "black"
+        if "linestyles" not in self.plot_args:
+            self.plot_args["linestyles"] = "solid"
```

Comment:
how about this instead ?
```suggestion
        self.plot_args = {
            "linestyles": "solid",
            **(plot_args or {})
        }
        if "cmap" not in plot_args:
            self.plot_args.set_default("colors", "black")
```

Reply:
No. `linestyles` is already being overriden correctly, let's keep the more verbose style only for when it's really needed.

Reply:
Note that the `if "linestyles" not in plot_args` condition is entirely redundant in your suggestion, too.